quill-jdbc/src/test/resources/application.conf

```diff
@@ -15,3 +15,6 @@ testPostgresDB.dataSource.serverName=${?POSTGRES_PORT_5432_TCP_ADDR}
 testH2DB.dataSourceClassName=org.h2.jdbcx.JdbcDataSource
 testH2DB.dataSource.url="jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'classpath:sql/h2-schema.sql'"
 testH2DB.dataSource.user=sa
+
+testSqliteDB.driverClassName=org.sqlite.JDBC
+testSqliteDB.jdbcUrl="jdbc:sqlite:./quill_test.db"
```

Comment:
The file is there, i have checked it. What about `testSqliteDB.jdbcUrl="jdbc:sqlite:/app/quill_test.db"`?